### PR TITLE
Connect cgi

### PR DESCRIPTION
**Pull request by Tatamize**
Now it seems to work with PHP, but it needs the same PHP environment. I will send you the php.ini of my environment.
It works with "config_file/normal.conf". please check the path to PHP and Python directory and modify the path in the config file if it's needed.

to show index ---->   http://127.0.0.1:6868/
to redirect  ----> http://127.0.0.1:6868/redirect
to upload a file  ----> http://127.0.0.1:6868/upload_php.html
to see the big image ----> http://127.0.0.1:6868/bigimage.html

you can see the second server is running   ----> http://127.0.0.1:6060

### CODE REVIEW

**Comment by Tatamize** [comment]
[php.ini_copy.txt](https://github.com/Zangetsu89/webserve/files/12621085/php.ini_copy.txt)

**Comment by Tatamize** [comment]
I sent the php.ini copy. The point is "variables_order". `variables_order = "GPCS"` this is the environment of Codam's Mac, normally "EGPCS" is useful but I can't change this setting because it needs sudo to change.
Check your PHP environment and check the variables_order in your php.ini.

my PHP Version => 7.1.33  (too old I think...but I couldn't install new version...)

**Comment by Tatamize** [comment]
I changed Makefile' s compile flag to -std=c++17 but it is not necessary, c++11 is also ok

**Comment by Tatamize** [comment]
I pushed the branch again, I moved the cpp files to the src directoy. The cleaning and checking is still going on.

To run the server,

change the Python and PHP path in the config_file/normal.conf.
run webserv, and access "localhost", it shows a page contains the links to other pages.

I tried to use Python to show another file and save the file data, but it needs time... (not easy...)
I made a simple Python page, not so interactive but it shows the current time.

File: www/server80/imgs/test.txt
Comment (Zangetsu89) [inline www/server80/imgs/test.txt:1]: text file in /imgs dir?

File: www/server5555/index.html
Comment (Zangetsu89) [inline www/server5555/index.html:1]: Dir name needs to be server5050 to match with the normal.conf file

**Review by Zangetsu89 — APPROVED** [review]
.DS files need to be removed.

File: www/server5555/index.html
Comment (Tatamize) [reply www/server5555/index.html:1]: Thanks, I will change it in the next version, I'm changing a lot, config file will be also modified.

**Pull request merged**